R/pkg/R/DataFrame.R

```diff
@@ -1869,14 +1871,22 @@ setMethod("where",
 #' path <- "path/to/file.json"
 #' df <- read.json(path)
 #' dropDuplicates(df)
+#' dropDuplicates(df, "col1", "col2")
 #' dropDuplicates(df, c("col1", "col2"))
 #' }
 setMethod("dropDuplicates",
           signature(x = "SparkDataFrame"),
-          function(x, colNames = columns(x)) {
-            stopifnot(class(colNames) == "character")
-
-            sdf <- callJMethod(x@sdf, "dropDuplicates", as.list(colNames))
+          function(x, col, ...) {
```

Comment:
How about such logic:
```
function(x, ...) {
  cols <- list(…)
  if (length(cols) == 0) {
  } else {
   if (!all(lapply(cols, function(col) { is.character(col) })) { stop }
   col <- cols[[1]]
   if (length(col) > 1) {…}
   else {…}	
  }
}
```
However, I have no strong opinion. 

A little problem of the current implementation is it does not check all (...) are character